proto/executor.proto

```diff
@@ -207,6 +207,20 @@ message ExecutorExecutionSummary {
 
 	// The execution concurrency for this executor
 	optional uint64 concurrency = 5;
+
+	oneof DetailInfo {
+		TableScanContext table_scan_context = 6;
+	}
+}
+
+message TableScanContext {
```

Comment:
May be we need to distinguish TiFlash and TiKV. As a TiKV member when I want to switch to this framework I don't know how to fill these fields.